jobserver/templates/_job_requests.html

```diff
@@ -11,21 +11,26 @@
         <th>Workspace</th>
         {% endif %}
         <th>Backend</th>
-        <th>Requested Action</th>
-        <th>Started At</th>
+        <th>Jobs finished</th>
+        <th>Started</th>
         <th>View jobs</th>
       </tr>
     </thead>
     <tbody>
       {% for group in page_obj %}
       <tr>
-        <td>
-          <div class="status status-icon {{ group.status }}"></div>
-          <span class="sr-only">{{ group.status }}</span>
+        <td style="width: 0">
```

Comment:
Does this hide the child `<div>`?

Reply:
Because tables are weird, it makes the `tbody > td` as small as possible given the content, which means the "Status" heading sets the width of the column.

Reply:
ah, "tables are weird" I can get behind!